include/eve/module/bessel/regular/cyl_bessel_i0.hpp

```diff
@@ -16,9 +16,11 @@ namespace eve
   template<typename Options>
   struct cyl_bessel_i0_t : elementwise_callable<cyl_bessel_i0_t, Options>
   {
-    template<eve::floating_value T>
-    EVE_FORCEINLINE constexpr
-    T operator()(T a) const noexcept { return EVE_DISPATCH_CALL(a); }
+    template<floating_value T>
+    EVE_FORCEINLINE constexpr T operator()(T a) const noexcept
+    {
+      return this->behavior(as<T>{}, eve::current_api, this->options(), a);
+    }
```

Comment:
What do these changes have to do with map?

Reply:
I'm all for the change, I am just not wild it being in the same diff.

Reply:
I'm not gonna make you split - but please don't do it in the future - it's extremely annoying to look through 500 files with different changes.